src/test/java/model/ladder/LadderTest.java

```diff
@@ -1,20 +1,65 @@
 package model.ladder;
 
 import static org.assertj.core.api.Assertions.assertThat;
+import static org.junit.jupiter.api.Assertions.assertAll;
+import static org.junit.jupiter.api.Assertions.assertEquals;
 
 import java.util.List;
+import model.bridge.Bridge;
+import model.gameResult.GameResult;
+import model.line.Line;
+import model.line.LineGenerator;
+import model.line.RandomLineGenerator;
 import model.player.Players;
+import model.prize.Prizes;
 import org.junit.jupiter.api.DisplayName;
 import org.junit.jupiter.api.Test;
 
 class LadderTest {
+    Players players = Players.of(List.of("pobi", "lala", "jojo", "gamja"));
+    Prizes prizes = Prizes.from(List.of("꽝", "식권" ,"꽝" ,"커피"), players);
+    LadderHeight ladderHeight = new LadderHeight(5);
+
     @DisplayName("사다리는 사다리 높이만큼의 라인을 가짐")
     @Test
     void testSizeOfLadderLines() {
-        Players players = Players.of(List.of("pobi", "lala"));
-        LadderHeight ladderHeight = new LadderHeight(5);
-        Ladder ladder = Ladder.of(ladderHeight, players);
+        LineGenerator lineGenerator = new RandomLineGenerator();
+        Ladder ladder = Ladder.of(ladderHeight, players, lineGenerator);
         assertThat(ladder.getLines().size())
                 .isEqualTo(5);
```

Comment:
이전 step 에서 작업해주신 내용인데, 이렇게 쓸 수도 있어요:

```suggestion
        assertThat(ladder.getLines())
                .hasSize(5);
```